tests/testthat/_snaps/databaseView/003.json

```diff
@@ -14,7 +14,7 @@
         "1.1.2"
       ],
       "score": [
-        0.3,
+        0.25,
```

Comment:
Do you know why this value changed for your PR? It hasn't changed on `dev`. Can you investigate why this is the case?

Reply:
I have no idea why this changed for me, but I can copy the 003.json from dev